build.gradle

```diff
@@ -19,20 +19,21 @@ repositories {
 
 allprojects {
     group = 'com.launchdarkly'
-    version = "1.0.1"
+    version = "2.0.0-SNAPSHOT"
     sourceCompatibility = 1.7
     targetCompatibility = 1.7
 }
 
 dependencies {
-    compile "org.apache.httpcomponents:httpclient:4.3.6"
-    compile "org.apache.httpcomponents:httpclient-cache:4.3.6"
-    compile "commons-codec:commons-codec:1.5"
-    compile "com.google.code.gson:gson:2.2.4"
+    compile "org.apache.httpcomponents:httpclient:4.5.2"
+    compile "org.apache.httpcomponents:httpclient-cache:4.5.2"
+    compile "commons-codec:commons-codec:1.10"
+    compile "com.google.code.gson:gson:2.6.2"
     compile "com.google.guava:guava:19.0"
-    compile "org.slf4j:slf4j-api:1.7.7"
+    compile "joda-time:joda-time:2.9.3"
```

Comment:
I was hesitant to bring in one more dependency, but Joda is more the gold standard for date/time in Java than java.util.Date